api/V1/Recording.js

```diff
@@ -84,11 +84,41 @@ module.exports = (app, baseUrl) => {
    * @apiSuccess {Number} recordingId ID of the recording.
    * @apiuse V1ResponseError
    */
+
   app.post(
-    apiUrl + "/:devicename",
+    apiUrl + "/device/:devicename/group/:groupname?",
```

Comment:
what does the `?` on the end mean?

Reply:
optional, so that it works with old bird-monitor releases